sdk.mjs

```diff
@@ -8,7 +8,7 @@ import path from 'node:path';
 
 /**
  * @typedef {import("./sdk.d.ts").ThirdPartyExtension} ThirdPartyExtension
- * @typedef {import("./sdk.d.ts").IconData} IconData
+ * @typedef {import("./types.js").IconData} IconData
```

Comment:
And here. I think that this PR should add these rewrites to importing all corresponding types from `.d.ts` files.